src/Library.js

```diff
@@ -1,7 +1,34 @@
+const searchShelf = require('./shelf').searchShelf;
+const unshelfBook = require('./shelf').unshelfBook;
 
```

Comment:
Keep this in mind for future endeavors. Maybe a better way to gain access to the whole file?